environment.yml

```diff
@@ -31,6 +31,6 @@ dependencies:
     - geovoronoi==0.4.0
     - celery==5.2.7
     - gunicorn==20.1.0
-    - git+https://github.com/xandercai/NewZeaLiDAR
+    - git+https://github.com/xandercai/NewZeaLiDAR/tree/v0.1
 
```

Comment:
I'm currently having a problem creating the environment with this new link here.

Reply:
It turns out that pip cannot install a git branch by ".../tree/branch" way; need to use "....git/@branch" way.
I have committed an update to the PR. Sorry about the mistake.

see [ref](https://stackoverflow.com/questions/20101834/pip-install-from-git-repo-branch)